### ISSUE TITLE
First announcement and Homework 1 description

### ISSUE THREAD

**Issue by aammd** [BIOL548O/Discussion#29]
Hello @BIOL548O/2017_students ! 

This is a "Github Issue". They are a handy way of keeping discussions together on github. The purpose of this particular issue is twofold:

1. This is an empty repository called Discussion. Any issue you post here can be read by anyone, and we can use this space to talk about questions related to the class. To attract someone's attention, write `@` followed by their username. If you want to ask me a question, use @aammd. If you want to talk to everyone, use @BIOL548O/2017_students .

2. A written explanation of the homework assignment: turn the README for your project into a brief description of your chosen data set. Try to practice some Markdown formatting techniques (for a full list see `Help > Markdown Quick Reference`). You can experiment with adding photographs or adding **bold** or _italic_ words. See [this project of mine](https://github.com/aammd/Predator_Phylogenetic_Diversity/blob/master/README.md) for some ideas ([raw markdown here](https://raw.githubusercontent.com/aammd/Predator_Phylogenetic_Diversity/master/README.md)) 
If you have any questions, feel free to get in touch -- perhaps below, or in a new issue!

Have a good weekend,
Andrew

**Comment by aesmith2** [comment]
Hi @aammd 

How can I add a dagger (unicode 2020) to my readme text? I couldn't find out how to do it. :(

**Comment by aesmith2** [comment]
Hey @BIOL548O/2017_students 
Has anyone been able to get the superscript command to work?

**Comment by aammd** [comment]
Hi @aesmith2 ,

Hm, i've never done that before! Looks like you can do [according to this website](http://www.fileformat.info/info/unicode/char/2020/index.htm)

*  `&#x2020;` which gives you &#x2020;
or
* ` 	&dagger;` which gives you  	&dagger;

this is a more general point that all @BIOL548O/2017_students might be interested in: Markdown is merely an easy way to write HTML, therefore any and all HTML can be used in a markdown document. That means that if you want to do something a little bit beyond the very basic formatting that Markdown allows, you may resort to HTML to solve it (e.g. the two solutions above are HTML encodings for the dagger symbol)

**Comment by aammd** [comment]
hi @aesmith2 et al, 

tl;dr : that superscript command (`basetosquare^2^`) won't work. 

Superscripts and subscripts are not going to work in your README files. This is because there are in fact several dialects of Markdown, and not all support every feature. For example, so-called [Github Flavoured Markdown](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) does not have superscripts. 

The markdown described in Rstudio's Help dialog is a related, but distinct, dialect of Markdown: this is ["Pandoc markdown"](http://pandoc.org/MANUAL.html#pandocs-markdown), named for [pandoc](http://pandoc.org/), the powerful document-conversion software. Pandoc is already installed on all (most?) of your laptops, because it is installed with an R package called `rmarkdown`. Pandoc is part of the magic inside the famous [rmarkdown reproducible documents](http://rmarkdown.rstudio.com/). It has a more flexible markdown, that allows authors to write very sophisticated documents -- even scientific papers.

A long-winded answer! But believe me, there is a lot of discussion about Markdown dialects. I felt you deserved to know the truth.

[obligatory xkcd](https://xkcd.com/927/)

**Comment by aesmith2** [comment]
I was able to get superscript to work by using < sup >entertexthere</ sup > as the code (minus the spaces between the < and sup>). Yahoooo!
I was not, however, able to get the codes you suggested for the dagger to work, so I did a work-around.

**Comment by aammd** [comment]
@BIOL548O/2017_students I forgot to add an important basic detail: the homework is due by the start of class on Tuesday.

**Comment by torhanssonfrank** [comment]
Hi @BIOL548O/2017_students. When I upload a picture it is lying down instead of standing. Has anyone figured out how to turn pictures?

**Comment by madelynore** [comment]
@BIOL548O/2016_students 
Re: pictures
I've also been having trouble getting captions on my pictures

![caption] (image.jpg) in Rstudio gives me the picture in github, but not the caption...

**Comment by rstorlund** [comment]
@torhanssonfrank I think there might be a default to upload the pictures in landscape orientation. You might have to change the size of the image to get it to upload in portrait orientation. Or crop them if possible. Just a thought...

**Comment by rstorlund** [comment]
@moodle2 to add a caption to your picture, just write the caption text on the next line of your readme.md. The text in the [] is alt text which shows up when your picture doesn't - it won't appear as a caption.

**Comment by torhanssonfrank** [comment]
Alright, thanks :) @rstorlund
